wandb/sdk/internal/internal_api.py

```diff
@@ -1228,7 +1228,11 @@ def agent_heartbeat(self, agent_id, metrics, run_states):
             )
         except Exception as e:
             # GQL raises exceptions with stringified python dictionaries :/
-            message = ast.literal_eval(e.args[0])["message"]
+            message = e.args[0]
+            try:
+                message = ast.literal_eval(message)["message"]
+            except SyntaxError:
+                pass
```

Comment:
what raises this?

Reply:
> # GQL raises exceptions with stringified python dictionaries :/

There was one case where it returns just a string, which would cause the `literal_eval` to fail.